test/e2e/antreapolicy_test.go

```diff
@@ -1657,33 +1657,42 @@ func testBaselineNamespaceIsolation(t *testing.T) {
 		SetPriority(1.0).
 		SetAppliedToGroup([]ACNPAppliedToSpec{{NSSelector: map[string]string{"ns": getNS("x")}}})
 	builder.AddIngress(ProtocolTCP, &p80, nil, nil, nil, nil, nil, nil, nil, nil, nil,
-		nil, nil, []metav1.LabelSelectorRequirement{nsExpOtherThanX}, nil,
+		nil, nil, nil, []metav1.LabelSelectorRequirement{nsExpOtherThanX},
 		nil, nil, crdv1beta1.RuleActionDrop, "", "", nil)
 
-	// create a K8s NetworkPolicy for Pods in namespace x to allow ingress traffic from Pods in the same namespace,
-	// as well as from the y/a Pod. It should open up ingress from y/a since it's evaluated before the baseline tier.
+	reachability := NewReachability(allPods, Connected)
+	reachability.ExpectNamespaceIngressFromNamespace(getNS("x"), getNS("y"), Dropped)
+	reachability.ExpectNamespaceIngressFromNamespace(getNS("x"), getNS("z"), Dropped)
+
+	// create a K8s NetworkPolicy for the x/a Pod to allow ingress traffic from Pods in the same namespace,
+	// as well as from the y/a Pod. It should open up ingress from y/a and deny ingress from y/b y/c based on
+	// Kubernetes NetworkPolicy default isolation model, since it's evaluated before the baseline tier.
```

Comment:
I think this won't validate the 3rd point: KNP default isolation also enforced before baseline ACNP.
Even if the implementation is the opposite, the test can still pass.
My suggestion was to remove ns-x from the rule, so only y/a is expected to access x/a, proving it's enforced before baseline ACNP.